### ISSUE TITLE
Find in reference for size() opens StringList.size()

### ISSUE THREAD

**Issue by danomatika** [processing/processing#4224]
Just a small thing, but I keep getting bitten by it when demoing "Find in reference".

Highlighting size() and choosing "Find in reference" opens the page for the StringList size() member function, not the P5 size() function.

Tested in P5 3.0.1

**Comment by Akarshit** [comment]
The cause of this is that in the keywords.txt file there are many occurrences of size like these
```
size 	FUNCTION2	FloatDict_size_
size	FUNCTION2	FloatList_size_
size	FUNCTION2	IntDict_size_  
size	FUNCTION2	IntList_size_  
size	FUNCTION2	JSONArray_size_  
size	FUNCTION1	size_  
size	FUNCTION2	StringDict_size_  
size	FUNCTION2	StringList_size_
```
So when constructong the map of the keywords to the html files the value of the size keyword gets repeatedly overwritten and is set to StringList_size at last. That's why the reference for StringList:size() is opened when opening reference for size().

A solution could be to take account of the class along with the method while searching for reference.

**Comment by dhowe** [comment]
This still exists and happens with any methods on objects with the same name as methods on PApplet, so filter() leads to PImage.filter(), etc (which is pretty confusing for beginners)

Tested in P5 3.2.1

**Comment by benfry** [comment]
Just fixed for 3.2.2 to save the beginners, though the larger issue remains: https://github.com/processing/processing/issues/4655
